api/src/shop/stripe_util.py

```diff
@@ -2,12 +2,134 @@
 from decimal import Decimal
 import random
 import time
+from logging import getLogger
 from typing import Callable, Optional, TypeVar
 
 from service.error import InternalServerError
-from shop.stripe_constants import STRIPE_CURRENTY_BASE
+from shop.models import Product
+from shop.stripe_constants import (
+    STRIPE_CURRENTY_BASE,
+    PriceType,
+    CURRENCY,
+)
 import stripe
 
+logger = getLogger("makeradmin")
+
+CURRENCY = "sek"  # TODO fix this
+
+
+# TODO cache?
+def get_stripe_product(makeradmin_product: Product, livemode: bool = False) -> stripe.Product | None:
+    id = f'prod_{makeradmin_product["id"]}' if livemode else f'test_{makeradmin_product["id"]}'
```

Comment:
Why are you indexing the products using strings? They are typed objects, so you should be able to use dot syntax.